clusterloader2/run-e2e.sh

```diff
@@ -29,14 +29,34 @@ export AZUREDISK_CSI_DRIVER_VERSION="v1.15.0"
 export AZUREDISK_CSI_DRIVER_INSTALL_URL="https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/${AZUREDISK_CSI_DRIVER_VERSION}/deploy/install-driver.sh"
 export WINDOWS_USE_HOST_PROCESS_CONTAINERS=true
 
-# Deploy the GCP PD CSI Driver if required
+# Deploy the GCE PD CSI Driver if required
 if [[ "${DEPLOY_GCI_DRIVER:-false}" == "true" ]]; then
    if [[ -z "${E2E_GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
-      echo "Env var E2E_GOOGLE_APPLICATION_CREDENTIALS must be set to deploy driver"
```

Comment:
We've lost this old message about application credentials. Is that not relevant anymore?

Reply:
Also - is there any way to distinguish between the case when user forgot credentials and when forgot yq? Current message could be quite confusing.

Reply:
Added informative message about the lack of credentials.